src/order/complete_lattice.lean

```diff
@@ -216,6 +216,12 @@ is_glb_univ.Inf_eq
 @[simp] theorem Inf_insert {a : α} {s : set α} : Inf (insert a s) = a ⊓ Inf s :=
 ((is_glb_Inf s).insert a).Inf_eq
 
+theorem Sup_le_Sup' (h : s ⊆ (insert ⊥ t)) : Sup s ≤ Sup t :=
+le_trans (Sup_le_Sup h) (le_of_eq (trans Sup_insert bot_sup_eq))
+
+theorem Inf_le_Inf' (h : s ⊆ (insert ⊤ t)) : Inf t ≤ Inf s :=
```

Comment:
Idem, I know the longer names look ugly, but otoh, readers will be happy about the extra info.